src/content/docs/en/reference/errors/locals-not-serializable.mdx

```diff
@@ -13,9 +13,7 @@ import DontEditWarning from '~/components/DontEditWarning.astro'
 <DontEditWarning />
 
 
-> **LocalsNotSerializable**: 
-            return `The information stored in `Astro.locals` for the path "HREF" is not serializable.\nMake sure you store only serializable data.`;
-         (E03034)
+> **LocalsNotSerializable**: The information stored in `Astro.locals` for the path "HREF" is not serializable. Make sure you store only serializable data. (E03034)
```

Comment:
Small tweak. Remembered we use code backticks for variables in these message examples:
```suggestion
> **LocalsNotSerializable**: The information stored in `Astro.locals` for the path "`HREF`" is not serializable. Make sure you store only serializable data. (E03034)
```